src/index.html

```diff
@@ -15,8 +15,86 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          class="img"
+          src="./images/logo.png"
+          alt="MOYO LOGO"
+        />
+      </a>
+      <nav class="nav">
+        <a
+          href="#"
+          class="box box-1 is-active"
+        >
+          Apple
+        </a>
+
+        <a
+          href="#"
+          class="box box-2"
+        >
+          Samsung
+        </a>
+
+        <a
+          href="#"
+          class="box box-3"
+        >
+          Smartphones
+        </a>
+
+        <a
+          href="#"
+          class="box box-4"
+          data-qa="hover"
+        >
+          Laptops & Computers
+        </a>
+
+        <a
+          href="#"
+          class="box box-5"
+        >
+          Gadgets
+        </a>
+
+        <a
+          href="#"
+          class="box box-6"
+        >
+          Tablets
+        </a>
+
+        <a
+          href="#"
+          class="box box-7"
+        >
+          Photo
+        </a>
+
+        <a
+          href="#"
+          class="box-8"
```

Comment:
Inconsistent class naming: The class for this link is 'box-8', whereas the previous links follow the pattern 'box box-x'. Consider renaming it to 'box box-8' for consistency.